ubuntu/16.04/usr/local/share/bootstrap/common_functions.sh

```diff
@@ -39,9 +39,15 @@ get_user_home_directory() {
 }
 
 as_user() (
-  set +x
```

Comment:
Can we reinstate this now that debugging is over, with a `set -x` on between lines 45 and 46

Reply:
I'd discussed this with @sroze before while we were pairing on it, but I agree now with @kierenevans

Reply:
Done 👍